### PR TITLE
feat: Add instructions to get MATIC

### PR DESCRIPTION
**Pull request by fzavalia**
- Removed extra "
- Added instructions to get MATIC

### CODE REVIEW

File: _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md [hunk 1]
```diff
@@ -35,3 +35,13 @@ The free transaction limit renews every day, so you can try again tomorrow. Alte
...
 
 Gas fees are variable.
+
+## Were can I get MATIC to pay for transaction fees?
+
+One way to obtain MATIC is buying it through https://account.decentraland.org/.
+
+Simply click on the BUY button in the Polygon MANA section, and change the crypto you want to purchase from MANA to MATIC.
+
+Another way is trhough Decentralized or Centralized exchanges. You can check https://polygon.technology/matic-token/ to see which exchanges operate with MATIC. It is important that if you want to use a Centralized exchange, make sure that said exchange allows withdrawals through the Polygon network.
```
Comment (nachomazzara) [inline _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md]:
```suggestion
Another way is through Decentralized or Centralized exchanges. You can check https://polygon.technology/matic-token/ to see which exchanges operate with MATIC. It is important that if you want to use a Centralized exchange, make sure that allows withdrawals through the Polygon network.
```

File: _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md [hunk 2]
```diff
@@ -35,3 +35,13 @@ The free transaction limit renews every day, so you can try again tomorrow. Alte
...
+
+If you have MATIC in the Ethereum network, you can always use the [Polygon Bridge](https://wallet.polygon.technology/bridge/) to transfer that MATIC to the polygon network.
```
Comment (nachomazzara) [inline _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md]:
```suggestion
If you have MATIC in the Ethereum network, you can always use the [Polygon Bridge](https://wallet.polygon.technology/bridge/) to deposit that MATIC to the polygon network.
```

File: _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md [hunk 3]
```diff
@@ -35,3 +35,13 @@ The free transaction limit renews every day, so you can try again tomorrow. Alte
 The MATIC fee will be deducted automatically as part of the transaction. You only need to be connected to the Polygon network and have MATIC in your wallet.
 
 Gas fees are variable.
+
+## Were can I get MATIC to pay for transaction fees?
+
+One way to obtain MATIC is buying it through https://account.decentraland.org/.
```
Comment (nicosantangelo) [inline _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md]: Maybe we can add `[the Account dapp](https://account.decentraland.org/) ` ?

File: _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md [hunk 4]
```diff
@@ -35,3 +35,13 @@ The free transaction limit renews every day, so you can try again tomorrow. Alte
...
+
+Simply click on the BUY button in the Polygon MANA section, and change the crypto you want to purchase from MANA to MATIC.
```
Comment (nicosantangelo) [inline _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md]:
```suggestion
Simply click on the BUY button in the Polygon MANA section, and exchange the crypto you want to purchase from MANA to MATIC.
```

File: _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md [hunk 5]
```diff
@@ -35,3 +35,13 @@ The free transaction limit renews every day, so you can try again tomorrow. Alte
...
+
+Another way is through Decentralized or Centralized exchanges. You can check https://polygon.technology/matic-token/ to see which exchanges operate with MATIC. It is important that if you want to use a Centralized exchange, make sure that it allows withdrawals through the Polygon network.
```
Comment (nicosantangelo) [inline _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md]: Maybe, to make it more concise:

```suggestion
Another way is through Decentralized or Centralized exchanges. You can check https://polygon.technology/matic-token/ to see which exchanges operate with MATIC. If you want to use a Centralized exchange, make sure that it allows withdrawals through the Polygon network.
```

**Review by nicosantangelo — COMMENTED** [review]
ausum

**Review by nachomazzara — APPROVED** [review]

File: _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md (hunk 3 above)
Comment (fzavalia) [reply _posts/blockchain-integration/2022-01-25-transactions-in-polygon.md]: Done

**Pull request merged**